### ISSUE TITLE
[BUG] add dataset / add protocol now only supports directories not files

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/ARCitect#83]
Somewhen during fixing this issue #47, the import function was broken for windows. 
The users could now only import directories, not single files anymore.

**Comment by Freymaurer** [comment]
Please add some steps on how to reproduce 😄

**Comment by Brilator** [comment]
@SabrinaZander, can you do me the favor and test adding "datasets"? I don't have a windows pc.

**Comment by SabrinaZander** [comment]
Version 0.16

**Comment by Freymaurer** [comment]
I verified this issue and found the following. The framework we use can only do "choose directory" (complete folders) or "choose file". I switched to "choose file", while also allowing to select multiples. If it is required to be able to select folders i could add a `+ Add Dataset Folder` option below/next to it.

tl;dr. Selecting folders and files with one button is "not possible". I decided for select file(s).

Let me know what you think @Brilator @SabrinaZander
